Affiliate program.md

```diff
@@ -0,0 +1,25 @@
+Below is what I managed to prepare for a few hours with the ChatGPT and my previous research.
+
+The topic needs further discussion.
+
+**Mechanism for creating value:**
+- Companies (AWS, Notion, etc) pay affiliate commission to partners for attracting clients. The commission is from 5% to 70% of the purchase amount.
+- Ubiquity can become a partner of such companies and transfer affiliate commission to bounty hunters.
+- For this bounty, hunters must pay with tokens.
+ The token can be UBQ or a new one - UXP (Ubiquity Experience), KARMA (karma of the developer), etc.
+
+**Mechanism for obtaining tokens:**
+- As a bonus for the efficient completion of tasks by bounty hunters.
+- Other options will be determined later.
+
+**Mechanism for spending tokens:**
+- Before buying a company product (AWS, Notion, etc), the bounty hunter transfers the appropriate amount of tokens to Ubiquity.
+- The purchase is made through the Ubiquity affiliate link.
+- After receiving the affiliate commission, Ubiquity transfers its amount to the bounty hunter.
+
+**Fun part**
+For the program, we can use the domain name with an allegory on GitHub - GiftHub.lol or GiftHub.dev
+
+**Summary**
+Essentially, a bounty hunter gets a discounts in exchange of our tokens.
+Such a scheme has no direct cost to Ubiquity and creates utility for tokens.
```

Comment:
seems like an easy ponzu, could be better is keeping a small amount is viable, not too hopeful on this because typically payouts from affiliates are really low 

 we can do some projections to determine a theoretical amount of revenue we could get from this for a given userbase